core/includes/config.inc

```diff
@@ -454,7 +454,10 @@ class Config {
 
     $this->name = $name;
     $this->storage = $storage;
-    $this->overrides = isset($config[$name]) ? $config[$name] : NULL;
+    // If $config is not an array some library or module may have interacted with the $config in a preboot situation.
```

Comment:
@seamuslee001 now thinking some more it should probably be like this:

```
$this->overrides = is_array($config) && isset($config[$name]) ? $config[$name] : NULL;
```

We want `$this->overrides` to be set to NULL at least.

Reply:
made the change now @herbdool